astroquery/cadc/cadctap/core.py

```diff
@@ -0,0 +1,411 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=============
+TAP plus
+=============
+
+"""
+from astroquery.utils.tap.core import TapPlus
+from astroquery.cadc.cadctap.tapconn import TapConnCadc
+from astroquery.cadc.cadctap.jobSaxParser import JobSaxParserCadc
+import requests
+import os.path
+from astropy.extern.six.moves.urllib.parse import urlencode
+
+try:
+    # python 3
+    import http.client as httplib

```

Comment:
We try to use python3 syntax everywhere, but then it's even more clear if this is imported as `from six.moves import http_client`